__tests__/app_test_.js

```diff
@@ -285,4 +284,91 @@ describe("GET /api/reviews/:review_id/comments - Error Handling", () => {
 });
 
   
-  
\ No newline at end of file
+describe("POST /api/reviews/:review_id/comments", () => {
+  test("201: Creates a new comment and responds with the posted comment", () => {
+    const comment = { username: "bainesface", body: "One of my favourie games!" };
+    return request(app)
+      .post("/api/reviews/4/comments")
+      .send(comment)
+      .expect(201)
+      .then(({ body: { postedComment } }) => {
+        expect(postedComment).toEqual(
+          expect.objectContaining({
+            comment_id: 7,
+            review_id: 4,
+            author: "bainesface",
+            body: "One of my favourie games!",
+            created_at: expect.any(String),
+            votes: 0,
+          })
+        );
+      });
+  });
+});
+describe("POST Error Handling", () => {
+  test("400: Responds with 'Bad request' error message when path is invalid", () => {
```

Comment:
Im being a little pedantic here but the test description is a little off in my opinion, I would probably say when given an invalid ID